### PR TITLE
Changes to support FAAS-647 - set the buttonSettings for the form pro…

### PR DESCRIPTION
**Pull request by stephanie-graham**
…grammatically, and do not override in the individual panels

### CODE REVIEW

File: src/modules/form/fheo.tsx [hunk 1]
```diff
@@ -31,6 +31,12 @@ export const Fheo = () => {
                                 'previous': 'GO BACK',
                                 'submit': 'SUBMIT MY COMPLAINT'
                             }
+                        },
+                        buttonSettings: {
+                            showPrevious: true,
+                            showNext: true,
+                            showCancel: false,
+                            showSubmit: true
                         }
```
Comment (Wube-T-Kifle) [inline src/modules/form/fheo.tsx:40]: this setting can be done in formio since, we want less formio code in the react app

Comment (Wube-T-Kifle) [reply src/modules/form/fheo.tsx:40]: Screen Shot 2021-02-17 at 8 37 10 AM

Comment (stephanie-graham) [reply src/modules/form/fheo.tsx:40]: Unfortunately, no, we can't set this in form.io if we want it to work.  The only way the button navigation works is if we do NOT set this setting on each individual panel.  Apparently the panel settings override the settings if we try to set them with code (like when the edit button is clicked on the review page).  So, we have to remove ALL the panel navigation button settings in form.io from every panel.  And the only way to have the "cancel" button removed on form display is to do what you see above.

I know what you are saying about removing code from the react app in favor of inline embedding.  We *should* be able to set both of the above options (buttonSettings and i18n) using inline embedding, so I am not too worried about this being something we could not recreate in an inline embed scenario.

Comment (stephanie-graham) [reply src/modules/form/fheo.tsx:40]: removing the panel navigation was the recommendation from form.io

**Review by Wube-T-Kifle — APPROVED** [review]

**Pull request merged**